apix-impl/build.gradle

```diff
@@ -47,6 +50,20 @@ allprojects {
         testImplementation 'org.alfresco:alfresco-remote-api'
         testImplementation 'org.alfresco:alfresco-data-model'
         testImplementation 'org.mockito:mockito-core'
+        // JUnit 5 dependencies
+        testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.1'
+        testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.1'
+        testImplementation 'org.junit.jupiter:junit-jupiter-params:5.9.1'
+        // Hamcrest dependencies
+        testImplementation 'org.hamcrest:hamcrest:2.2'
+    }
+
+    test {
+        // Always run unit tests
+        outputs.upToDateWhen { false }
```

Comment:
Is that still necessary?

Reply:
i dont see the use of it. Running the docker-build which would trigger this, seems to generate empty files.
Removed it from build